meson.build

```diff
@@ -3942,6 +3971,7 @@ foreach tuple : [
         ['rfkill'],
         ['sysext'],
         ['systemd-analyze',       conf.get('ENABLE_ANALYZE') == 1],
+        ['sysupdate'],
```

Comment:
shouldn't this be `['sysupdate', conf.get('ENABLE_SYSUPDATE') == 1],`?

Reply:
no need. the second field only needs to be specified if the ENABLE_XYZ/HAVE_XYZ name is not directly generated from the component name. for systemd-analyze that's the case, because the "systemd-" prefix needs to be suppressed. But for sysupdate it's fine. since we call the component "sysupdate" and the condition ENABLE_SYSUPDATE, so we are good, no "systemd-" prefix to strip anywhere.